application/src/main/java/org/thingsboard/server/actors/calculatedField/CalculatedFieldManagerMessageProcessor.java

```diff
@@ -257,6 +256,8 @@ private void onProfileDeleted(ComponentLifecycleMsg msg, TbCallback callback) {
     }
 
     private void onTenantProfileUpdated(ComponentLifecycleMsg msg, TbCallback callback) {
+        cancelReevaluationTask();
+        scheduleCfsReevaluation();
```

Comment:
Please do that only if the cfCheckInterval is changed. Most times it won't be, so we shouldn't disrupt the reeval on any update